### PR TITLE
Telemetry site

### PR DESCRIPTION
**Pull request by i-jey**
REST API + simple script to upload data.

### CODE REVIEW

File: data_viz/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\Owner\\envs\\ubcsolar\\Scripts\\python.exe"
```
Comment (cng123) [inline data_viz/.vscode/settings.json]: Moving forward, should we leave files like these untracked?

File: data_viz/send_data.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+'''
+    A quick script to send data as post requests
+'''
+
+import requests 
+
+url = "http://localhost:8000/data_viz/api/nodes/"
+data1 = "CellC - 680mA DischCurr.xlsx"
```
Comment (cng123) [inline data_viz/send_data.py:8]: What is this for? WIll we be using this document?

File: data_viz/telemetry_site/.vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\Owner\\envs\\ubcsolar\\Scripts\\python.exe"
```
Comment (cng123) [inline data_viz/telemetry_site/.vscode/settings.json]: Same as other comment about vscode configurations

**Comment by cng123** [comment]
In addition to the comments above: 
- Should we leave cached python files untracked?
- Is the sqllite db solely for alumni night or would we be using it for other purposes in the future?

File: data_viz/.vscode/settings.json (hunk 1 above)
Comment (i-jey) [reply data_viz/.vscode/settings.json]: We should, good catch.

File: data_viz/send_data.py (hunk 2 above)
Comment (i-jey) [reply data_viz/send_data.py:8]: Yes, for the alumni event. Laila sent me this data and I figured we could run a 'pseudo-realtime' trial with some real data.

File: data_viz/telemetry_site/.vscode/settings.json (hunk 3 above)
Comment (i-jey) [reply data_viz/telemetry_site/.vscode/settings.json]: Agreed!

**Comment by i-jey** [comment]
1. Cached python files should be left untracked - I can write up a quick .gitignore to make sure I don't goof up again! 
2. sqlite db is just for the alumni event, if we do decide to use the site for real, we would probably switch to postgresql/mysql or something similar.

**Review by cng123 — APPROVED** [review]
Everything looks good

**Pull request merged**